config/webhook/manifests.yaml

```diff
@@ -9,7 +9,7 @@ webhooks:
   clientConfig:
     service:
       name: webhook-service
-      namespace: $(mesh-namespace)
+      namespace: system
```

Comment:
Can you revert this? It doesn't look related.

Reply:
Yeah. it is updated every time the manifests are generated.
@Jooho fyi.

Reply:
@mholder6 can you please revert?